src/runtime/loader.ts

```diff
@@ -64,19 +64,21 @@ export function extractStack(
   requiredAPIs: ManifestRequiredAPI[],
   prefix = ''
 ) {
-  for (const [name, val] of Object.entries(module)) {
+  for (const [name, valAsUnknown] of Object.entries(module)) {
+    // We're introspecting untrusted code here. Any is appropraite
+    const val: any = valAsUnknown;
     if (
       typeof val === 'function' &&
-      val['__endpoint'] &&
-      typeof val['__endpoint'] === 'object'
+      (val as any).__endpoint &&
```

Comment:
nit* is `(val as any)` required here? I'd guess it's already any based on L69.

Reply:
You're right. I was doing this manually and then used the any cast above. Will remove.